react/package.json

```diff
@@ -189,5 +189,9 @@
     "engines": {
         "node": ">=14.17",
         "npm": ">=6.14"
+    },
```

Comment:
This is already done. I think we should aim for `"react": ^17 || ^18` but we should wait until we have completed migration to MUI 5.

Reply:
@hkfb Can we close this PR then?

Reply:
Yes.

Reply:
Yes.